hackerrank/src/Algorithms/Strings/PalindromeIndex.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Fedotkin.Dotnet.HackerRank.Algorithms.Strings.PalindromeIndex;
+
+public class Result
+{
+    public static int PalindromeIndex(string s)
+    {
+        string test = "";
+        int index = -1;
+        int length = s.Length;
+
+        for (int i = 0; i < length / 2; i++)
+        {
+            if (s[i] != s[length - 1 - i])
+            {
+                test = s.Remove(i, 1);
+                index = i;
+                break;
+            }
+        }
+
+        if (index == -1)
+            return index;
+
+        for (int i = 0; i < (length - 1) / 2; i++)
+        {
+            if (test[i] != test[length - 2 - i])
```

Comment:
```suggestion
            if (list[i] != list[length - 2 - i])
```
Мяркую, што можна абыйсціся без дадатковай зменнай `test`, бо хопіць і адной `list`